src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -261,6 +261,9 @@ public function locateResource($name, $dir = null, $first = true)
         $files = [];
 
         if ($isResource && file_exists($file = $dir.'/'.$bundle->getName().$overridePath)) {
+            if (0 === strpos($dir, 'app') || 0 === strpos($dir, 'src')) {
```

Comment:
`$dir` would normally be `%kernel.root_dir%/Resources`. I don't see when it would ever start with `app` or `src`? What is this supposed to do?